omero/sysadmins/limitations.rst

```diff
@@ -81,6 +81,21 @@ described above.
 
 .. _minmax_limitation:
```

Comment:
this now needs to be moved down i.e. just above 
``Calculation of minima and maxima pixel values``

Reply:
fixed in https://github.com/ome/omero-documentation/pull/2248/commits/76cf310518defd8569c6631bae89bdbb54346cef